jarbas/chamber_of_deputies/querysets.py

```diff
@@ -49,6 +51,10 @@ def tuple_filter(self, **kwargs):
     def was_ordered(self):
         return bool(self.query.order_by)
 
+    def search_vector(self, search_term):
+        query = SearchQuery(search_term, config='portuguese')
+        return self.filter(search_vector=query)
```

Comment:
In the name of DRY I would include [the rank](https://github.com/ricardochaves/serenata-de-amor/blob/eb22d465b487060c2c10b971397a08a58b9aa1f6/jarbas/dashboard/admin.py#L374-L376) here and use this queryset also in the `admin.py`. If you think this is too much for this PR, would you open a new PR so we can refactor this later?